scripts/common.py

```diff
@@ -422,6 +428,26 @@ def check_prs_for_current_release_are_released(repo):
                 'released before release summary generation.')
 
 
+def kill_processes_based_on_regex(pattern):
+    """Kill processes based on provided regex.
+
+    Args:
+        pattern: str. Pattern for searching processes.
+    """
+    regex = re.compile(pattern)
+    if PSUTIL_DIR not in sys.path:
+        sys.path.insert(1, PSUTIL_DIR)
+    import psutil
+    for process in psutil.process_iter():
+        try:
+            cmdline = ' '.join(process.cmdline())
+            if regex.match(cmdline) and process.is_running():
+                python_utils.PRINT('Killing %s ...' % cmdline)
+                process.kill()
+        except psutil.Error:
+            continue
```

Comment:
Shouldn't we print something, if this case isn't expected to happen? Or will this retry? (If so, please add a comment to indicate that.)

Reply:
In theory, the exception will not influence the actual behaviour. Exceptions would be threw include `AccessDenied`, `NoSuchProcess`, `ZombieProcess`, `TimeoutExpired`. Those exceptions can just be ignored. I will add a comment.

https://psutil.readthedocs.io/en/latest/#exceptions